.config/jest.config.js

```diff
@@ -21,7 +21,8 @@ async function createJestConfig() {
             ],
             reporters: [
                 `default`,
-                [`${HOME_DIR}/src/reporter.js`, {metrics}],
+                [ "jest-junit", {outputName: "junit.xml"} ],
```

Comment:
```suggestion
                [ "jest-junit", {outputName: `${HOME_DIR}/junit.xml`} ],
```
I think it's preferable to provide full path just as in `src/constants.js`

Reply:
make sense!